deployment/helm/node-feature-discovery/templates/clusterrole.yaml

```
@@ -25,6 +25,21 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  resourceNames:
+  - "nfd-master.nodefeaturediscoveries.nfd.kubernetes.io"
```

This needs to be renamed, too 
```suggestion
  - "nfd-master.nfd.kubernetes.io"
```

---

fixed!